### PR TITLE
Add files for GCP Marketplace packing

### PR DESCRIPTION
**Pull request by ooq**

### CODE REVIEW

File: marketplace/README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
+# Setting Up for Marketplace Solution
```
Comment (nenaddedic) [inline marketplace/README.md]: Can you qualify "Marketplace" so it is clear it's GCP Marketplace (or whatever the official name)?

File: marketplace/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+WORKDIR /go/src/github.com/grafeas/voucher
+COPY . .
+RUN apk --no-cache add \
+    git \
+    make && \
+    make voucher_server
+
+# Final build
+FROM alpine:3.8
```
Comment (nenaddedic) [inline marketplace/Dockerfile:12]: How does this interact with line 2?  I didn't even know you could 'FROM' two different images.

**Review by nenaddedic — APPROVED** [review]

File: marketplace/Dockerfile (hunk 2 above)
Comment (ooq) [reply marketplace/Dockerfile:12]: The first base image is used simply for building the voucher_server, and then everything is left out except for the voucher_server which is explicitly copied to the second base image.

File: marketplace/README.md (hunk 1 above)
Comment (ooq) [reply marketplace/README.md]: ack.

**Review by ooq — COMMENTED** [review]
Comments addressed.

**Pull request merged**